src/browserServerImpl.ts

```diff
@@ -43,19 +44,24 @@ export class BrowserServerLauncherImpl implements BrowserServerLauncher {
   }
 
   async launchServer(options: LaunchServerOptions = {}): Promise<BrowserServer> {
+    const forwardingProxyServer = await BrowserServerForwardingProxy.create(options._acceptForwardedPorts);
     // 1. Pre-launch the browser
     const browser = await this._browserType.launch(internalCallMetadata(), {
       ...options,
       ignoreDefaultArgs: Array.isArray(options.ignoreDefaultArgs) ? options.ignoreDefaultArgs : undefined,
       ignoreAllDefaultArgs: !!options.ignoreDefaultArgs && !Array.isArray(options.ignoreDefaultArgs),
       env: options.env ? envObjectToArray(options.env) : undefined,
+      ...forwardingProxyServer.browserLaunchConfig(),
     }, toProtocolLogger(options.logger));
+    forwardingProxyServer.registerBrowser(browser);
 
     // 2. Start the server
     const delegate: PlaywrightServerDelegate = {
       path: '/' + createGuid(),
       allowMultipleClients: true,
```

Comment:
Make this false when `_acceptForwardedPorts` is set.

Reply:
way better, than we can handle it there instead of manually. :+1: